adoc/release-4.2.1.adoc

```diff
@@ -14,8 +14,13 @@
 
 === Metrics Server
 
-The Metrics Server now supports monitoring of *CPU* and *memory* of a pod or node.  You can use commands `kubectl top nodes` get status for node and use `kubectl top pods` to get status for pod.
-This information can also be used and show in be Prometheus or Grafana.
+{productname} now supports the metrics server as an addon to monitors the *CPU* and *memory* of a pod or node.  You can use commands `kubectl top nodes` get status for node and use `kubectl top pods` to get status for pod. Also, with metrics-server default built-in by {productname}, the user could set up a horizontal pod autoscaler (HPA) to auto-scales the number of pods or set up a vertical pod autoscaler (VPA) to auto allocates more or less CPUs and memory resource.
+
+{productname} did not detect existed metrics-server installed by the administrator, so please remove the installed metrics-server manually before upgrade addons.
```

Comment:
```suggestion
{productname} did not detect existing metrics-server installed by the administrator, so please remove the installed metrics-server manually before upgrade addons.
```

This is missing the commands needed to remove this. I think it also should go into the "required actions" part of the notes.

Will not removing the old metrics-server break the functionality? If so we need to specifically add this to the upgrade instructions as well.

Reply:
> Will not removing the old metrics-server break the functionality? If so we need to specifically add this to the upgrade instructions as well.

I think yes, let me update this PR later~

Reply:
@r0ckarong updated! PTAL :bow: